src/locales/en/fields.yml

```diff
@@ -8,25 +8,33 @@
   label: Year
 
 surname:
-  label: Last Name
+  label: Last name
   hint:
   validation:
     default: "Enter your last name as it appears on your photo ID"
     maxlength: "Enter your last name as it appears on your photo ID"
-    regexName: "Enter your last name as it appears on your photo ID"
+    regexName: "Your last name cannot include numbers"
```

Comment:
need to update these to refer to `regexNumber` and `regexSpecialCharacters`

Reply:
🤦